resources/META-INF/plugin.xml

```diff
@@ -236,12 +236,12 @@
     <!-- main toolbar run actions -->
     <group id="Flutter.MainToolbarActions.Run">
       <separator/>
-      <action id="Flutter.Toolbar.ReloadAction" class="io.flutter.actions.ReloadFlutterAppRetarget"
-              description="Reload"
-              icon="FlutterIcons.HotReload">
-        <keyboard-shortcut keymap="$default" first-keystroke="ctrl alt SEMICOLON"/>
-        <keyboard-shortcut keymap="$default" first-keystroke="ctrl BACK_SLASH"/>
-      </action>
+      <!--<action id="Flutter.Toolbar.ReloadAction" class="io.flutter.actions.ReloadFlutterAppRetarget"-->
+              <!--description="Reload"-->
+              <!--icon="FlutterIcons.HotReload">-->
+        <!--<keyboard-shortcut keymap="$default" first-keystroke="ctrl alt SEMICOLON"/>-->
+        <!--<keyboard-shortcut keymap="$default" first-keystroke="ctrl BACK_SLASH"/>-->
+      <!--</action>-->
```

Comment:
Hmmm.  We do want it in both places though right?  In that case, can we replace one with an action reference?

    <reference ref="Flutter.Toolbar.ReloadAction"/>